ceph-ansible-prs/config/definitions/ceph-ansible-prs.yml

```diff
@@ -14,9 +15,27 @@
         - 'ceph-ansible-prs-auto'
 
 # tests that will not auto start when a PR is created, but
-# they can be requested with a trigger phrase
+# they can be requested with a trigger phrase. Run on smithi.
 - project:
-    name: ceph-ansible-prs-trigger
+    name: ceph-ansible-prs-trigger-smithi
+    slave_labels: 'vagrant && libvirt && smithi'
+    release:
+      - luminous
+    ansible_version:
+      - ansible2.3
+    scenario:
+      - purge_cluster
+      - update_docker_cluster
+      - switch_to_containers
+      - purge_docker_cluster
+    jobs:
+        - 'ceph-ansible-prs-trigger'
+
+# tests that will not auto start when a PR is created, but
+# they can be requested with a trigger phrase. Run on OVH.
+- project:
+    name: ceph-ansible-prs-trigger-ovh
+    slave_labels: 'vagrant && libvirt !smithi'
```

Comment:
I think this should be: ``vagrant && libvirt && !smithi``

Reply:
Good catch.  Fixed.